evaluators/text/JaccardSimilarityEvaluator/__init__.py

```diff
@@ -12,9 +12,9 @@ def metric(self):
         return 'JaccardSimilarity'
 
     def evaluate(self, actual: str, desired: str):
-        a = set(actual.lower().split())
-        b = set(desired.lower().split())
-        c = a.intersection(b)
-        total = len(a) + len(b) - len(c)
-        return float(len(c)) / total if total != 0.0 else 0.0
+        acutal_words = set(actual.lower().split())
+        desired_words = set(desired.lower().split())
+        intersection = acutal_words.intersection(desired_words)
+        union_length = len(acutal_words) + len(desired_words) - len(intersection)
+        return float(len(intersection)) / union_length if union_length != 0.0 else 0.0
```

Comment:
why is this `float` conversion required?

Reply:
You are right, it is not necessary in python.